src/components/header/renderer.tsx

```diff
@@ -159,12 +163,21 @@ class Header extends React.Component<Props, State> {
     });
     const responseGistCommits = await gistCommits.json();
     if (revision.length === 0) {
+      this.setState({
+        invalidFilename: false,
+        invalidRevision: false,
+      });
```

Comment:
Considering there is already an error in revision and now revision field is left empty we remove the error message. Same for filename. I could've left out the filename and changed it later but it is faulty for the case when the url is invalid and revision is empty and there was a filename error beforehand.